cypress/page/menu-content.page.ts

```diff
@@ -5,7 +5,7 @@ class MenuContentPage {
   constructor() {
       /* Now the selector searchs the last li in Ul and and a label
         that is inside the li */
-      this.tShirtMenu = "#block_top_menu > ul > li:last-child a";
+      this.tShirtMenu = "#block_top_menu > ul > li:last-child a[title='T-shirts']";
```

Comment:
remove the `:last-child` -> `"#block_top_menu > ul > li > a[title='T-shirts']"` should be fine